### ISSUE TITLE
Dev test Charges and Fields 1.0.0-dev.15

### ISSUE THREAD

**Issue by andrewadare** [phetsims/tasks#661]
Charges and Fields is ready for a test. Only a small number of minor issues have been worked on lately and it should be getting very stable.

**Testing Link**
http://www.colorado.edu/physics/phet/dev/html/charges-and-fields/1.0.0-dev.15/charges-and-fields_en.html

Please test on the following platforms:

- [x] iPad 2 latest iOS
- [x] Chromebook
- [x] Mac Safari
- [x] Windows 10 Edge

*The phet-io brand is not ready for testing--there has been a *lot* of churn there this week, so that will have to wait for another time, hopefully soon though*

**Comment by ariel-phet** [comment]
@phet-steele @brroberts1231 @alynne-phet please make this the current top sim testing priority.

**Comment by phet-steele** [comment]
@andrewadare @ariel-phet this testing is done. I'd recommend another dev version. Pretty significant changes will probably be needed, but it's your call too @andrewadare.

**Comment by andrewadare** [comment]
Agreed, another dev version is warranted. More bugs still to fix in the meantime.

**Comment by andrewadare** [comment]
@phet-steele @ariel-phet I just pushed dev.16 to spot. I worked on many bugfixes over the week, and a new testing round seems appropriate. 

The issue that occupied the most of my time is the one I made the least real progress on (Significant memory leak, https://github.com/phetsims/charges-and-fields/issues/83). I took some of @jbphet's time yesterday for an apprenticeship on profiling (which was quite useful), but had a very hard time definitively tracking lost resources back to C&F (mostly js builtins and scenery objects).

What's the protocol here--should I open a new issue in this repo for dev.16?
